testing/functional/tests/directLine.test.js

```diff
@@ -48,16 +54,16 @@ async function sendMessage(client, conversationId) {
 
 function getMessages(client, conversationId) {
     const watermark = null;
-    return client.Conversations.Conversations_GetMessages({
+    return client.apis.Conversations.Conversations_GetMessages({
         conversationId: conversationId,
         watermark: watermark,
     }).then((response) => {
         return response.obj.messages.filter((message) => message.from !== directLineClientName);
     });
 }
 
-function getConversationId(client) {
-    return client.Conversations.Conversations_NewConversation().then((response) => response.obj.conversationId);
+async function getConversationId(client) {
+    return client.apis.Conversations.Conversations_NewConversation().then((response) => response.obj.conversationId);
```

Comment:
If this is async, shouldn't we use the `await` operator?

Reply:
using the await here is optional, if the one calling the getConversationId has the await keyword, it will suffice.